requirements-test.txt

```diff
@@ -4,4 +4,5 @@ pytest-django
 coveralls
 Django>=1.9
 psycopg2
-opencivicdata-divisions
+-e git+https://github.com/california-civic-data-coalition/python-opencivicdata-divisions.git@HEAD#egg=opencivicdata-divisions
+backports.csv
```

Comment:
this dependency should be specified in opencivicdata-divisions if it is needed

& the switch to the CCDC version should be fixed after merging py-ocd-divisions but before this